### PR TITLE
outputTerminal: Add unit test for dimensions

### PR DESCRIPTION
**Pull request by SeaDve**

### CODE REVIEW

**Comment by SeaDve** [comment]
Closing this since it's apparently not possible to resize terminal using a command for this case.

**Comment by bilelmoussaoui** [comment]
> Closing this since it's apparently not possible to resize terminal using a command for this case.
> 
> We could test it, but command internals has to be exposed publicly.

I would be fine exposing things that are useful for tests only

**Comment by SeaDve** [comment]
I'll see the most minimal way to expose those.

**Comment by SeaDve** [comment]
This seems to work hehe :D

File: src/command.ts [hunk 1]
```diff
@@ -122,8 +122,9 @@ export class Command {
      * Spawn this with using node-pty
      * @param terminal Where the output stream will be sent
      * @param token For cancellation. This will send SIGINT on the process when cancelled.
+     * @returns the process
      */
-    spawn(terminal: OutputTerminal, token: CancellationToken): Promise<void> {
+    spawn(terminal: OutputTerminal, token: CancellationToken): Promise<pty.IPty> {
```
Comment (bilelmoussaoui) [inline src/command.ts:127]: I wonder if we should have a small interface to avoid making the pty.IPty as part of the "public api"

Comment (SeaDve) [reply src/command.ts:127]: I think it is not interruptive enough to have its own function, if that is what you mean by interface

Comment (bilelmoussaoui) [reply src/command.ts:127]: Oh I meant returning something like 
```typescript
class PseudoTerminal {
   private inner: pty.Ipty,

   dimensions() {
    return inner.dimensions
   }
}
```

Comment (SeaDve) [reply src/command.ts:127]: Ah, I see that will work, but not sure if it is worth it, since the pty is killed/exited when the `pty.IPty` was returned.

**Pull request merged**